generator/src/lib.rs

```diff
@@ -1499,7 +1500,7 @@ pub fn derive_debug(_struct: &vkxml::Struct, union_types: &HashSet<&str>) -> Opt
         let debug_value = if is_static_array(field) && field.basetype == "char" {
             quote! {
                 &unsafe {
-                    ::std::ffi::CStr::from_ptr(self.#param_ident.as_ptr() as *const c_char)
+                    ::std::ffi::CStr::from_ptr(self.#param_ident.as_ptr())
```

Comment:
I made sure this one is safe: `CStr::from_ptr` already takes a `c_char` and because `field.basetype == "char"` here the underlying type of `$param_ident` is already `[c_char; ...]`. Hence, no issues with `i8` vs `u8` when compiling on armv7/aarch64 :)